### PR TITLE
Free networks

### PR DESCRIPTION
**Pull request by wonderwomancode**
Added information to the networks page about gasless free claims and in general cleaned the page up a bit.

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
I think this page is autogenerated. Let me check with @clemsos

**Comment by julien51** [comment]
Confirmed https://github.com/unlock-protocol/unlock/blob/master/scripts/docs-networks-pr.sh
So I think we need to update the template there and to mark the "Free Claims" let's just add a `freeClaim` property in the networks package with the max amount we use for free claims as per https://github.com/unlock-protocol/unlock/blob/master/locksmith/src/utils/keyPricer.ts#L26

**Comment by julien51** [comment]
Also please add a `This page is autogen from ...` at the top of the template so that we don't try to change it again in the future.

**Comment by wonderwomancode** [comment]
@julien51 @searchableguy added the updates for the page to template for the auto-gen script. I reverted the networks page to what was in the master, but please approve the other changes needed for the auto-generated page to be styled properly.

**Review by searchableguy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "docusaurus-preset-openapi": "0.4.0",
     "prism-react-renderer": "1.3.5",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "react-icons": "^4.7.1"
```
Comment (julien51) [inline package.json:16]:
```suggestion
    "react-icons": "4.7.1"
```

**Review by julien51 — APPROVED** [review]

**Pull request merged**